guides/GuideForGuests.md

```diff
@@ -10,7 +10,7 @@ This guide should talk you through what you need to know to prepare for the call
 
 ---
 
-***tl;dr: join the google hangout 5 mins early, complete [this google form](https://goo.gl/forms/3foIuNP9G7kmCW193) in advance & have fun*** :sparkles:
+***tl;dr: test your google hangout connection in advance, complete [this google form](https://goo.gl/forms/3foIuNP9G7kmCW193) in advance, join the google hangout 5 mins early & have fun*** :sparkles:
```

Comment:
I think the suggestion to test your google hangout connection should lead to some sort of link, just so that guests who are less familiar with the software don't miss checking the mic (which I will imagine will be the source of most problems!). Maybe something like this: https://productforums.google.com/forum/#!topic/hangouts/DWVcoq7B7xk

Reply:
Thanks @emdupre & @adelavega. I've updated this tl;dr to now link to more details that are further down the document. I hope that makes sense 😸